### ISSUE TITLE
Ctrl+Letter does not seem to work on Azerty keyboard

### ISSUE THREAD

**Issue by Moeryn** [rhysd/NyaoVim#47]
I am starting to play around with NyaoVim and it works great :) expect for all Ctrl + letter action.

### Expected Behavior

Ctrl + v to vertically select

### Actual Behavior

Only v is catch by neovim

### Steps to Reproduce (including precondition)

No particular steps, simply open a file with some content.

### Screenshot on This Problem (if possible)

Here the output of Ctrl-v

![image](https://cloud.githubusercontent.com/assets/1199435/14530402/30ee62d6-0259-11e6-9b1e-3e3ecdf9ae90.png)

### Your Environment

- OS: Linux/fedora
- NyaoVim version: NyaoVim version 0.0.19
  electron : 0.37.5
  chrome : 49.0.2623.75
  node : 5.10.0
  v8 : 4.9.385.28
- `nvim` version: NVIM v0.1.4-23-gf47a20a
- Keyboard layout: azerty
- Linux: Window manager: awesome & KDE

**Comment by Moeryn** [comment]
On my other computer, Debian & Gnome NyaoVim works perfectly fine.

**Comment by samadivk** [comment]
I'm getting a similar issue on Manjaro with Colemak, except that, rather than treating, say, <c-y> as y as with @Moeryn's issue, I'm getting <c-o>, which is what it would've been if I were using QWERTY. This happens for all keys pressed with Ctrl.

**Comment by amagee** [comment]
Same issue here with Dvorak layout; issue goes away immediately if I change the layout to QWERTY -- so looks like it's a general issue with non-QWERTY layouts?

**Comment by amagee** [comment]
Same issue here with Dvorak layout; issue goes away immediately if I change the layout to QWERTY -- so looks like it's a general issue with non-QWERTY layouts?

**Comment by amagee** [comment]
Did a bit of digging. Turns out that in the 'keydown' handler `NeovimInput.prototype.onInputNonText`, `event.key` is already wrong. This seems relevant: https://github.com/atom/atom-keymap/issues/37

**Comment by denysvitali** [comment]
I'm having a similar problem with Atom installed via AUR. The Z / Y keys are inverted (I use a QWERTZ keyboard). When I type "z" in Atom the character is displayed correctly (a "z" apperas), but when I use a shortcut it is inverted.

**Comment by RMKD** [comment]
It seems this can be traced to using a later version of Electron: https://github.com/atom/atom/issues/13170 - I am seeing the Ctrl + dvorak error on Manjaro with electron 1.4.12-1 installed. 

I solved by uninstalling the community edition and installing atom-editor-bin from AUR (which bundles the correct version of electron) https://bugs.archlinux.org/task/50357

**Comment by Reylak** [comment]
Just to say that update 1.4.12-2 of electron solved the problem of wrong keymap for Ctrl+<> bindings.

For ArchLinux users, this is now the [version available from official community repo](https://www.archlinux.org/packages/community/x86_64/electron/) for (compiled on 17/12/2016).

**Comment by denysvitali** [comment]
Will check, thanks @Reylak !

**Comment by amagee** [comment]
Yep, I was able to fix the problem by editing the NyaoVim's `package.json` file, replacing the `electron-prebuilt` version to `"*"` and doing an `npm upgrade`.
